src/app/workflow-dialog/workflow-dialog.component.spec.ts

```diff
@@ -0,0 +1,29 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { WorkflowDialogComponent } from './workflow-dialog.component';
+import {MatDialogModule} from '@angular/material/dialog';
+
+describe('WorkflowDialogComponent', () => {
+  let component: WorkflowDialogComponent;
+  let fixture: ComponentFixture<WorkflowDialogComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [ WorkflowDialogComponent ],
+      imports: [
+        MatDialogModule,
```

Comment:
you need to include matDialog in some of your tests,  I added it here, but I think we are still missing some somewhere else.  PLEASE RUN THE TESTS before creating a pull request.